pandas/core/series.py

```diff
@@ -1335,8 +1336,8 @@ def idxmax(self, axis=None, skipna=True, *args, **kwargs):
         return self.index[i]
 
     # ndarray compat
-    argmin = idxmin
-    argmax = idxmax
+    argmin = deprecate('argmin', idxmin)
+    argmax = deprecate('argmax', idxmax)
```

Comment:
That's fine for me as well, if that works out. Probably the simplest case is to add something to the beginning of the existing docstring.